dotcom-rendering/docs/architecture/029-only-esmodules.md

```diff
@@ -0,0 +1,21 @@
+# Only serve JavaScript ES Modules
+
+## Context
+
+As of June 2022, the Guardian no longer supports IE11.
+
+[cut the mustard]: ./asdf.md
+
+## Decision
+
+Chrome, Edge, Firefox, Opera, Safari all support ES Modules, so we can stop
+serving JavaScript for browsers that don’t [cut the mustard][].
```

Comment:
Same as above

Reply:
I think it's going to be helpful to flag this change to future devs as best as possible but just to say that these [decisions docs](https://adr.github.io/) are meant as historical records and - I think - they are not meant to be edited.

I've actually violated that principle myself before but after reading up on this maybe I was wrong to do this. As I understand it, the correct process is to create a new document to record the change in thinking

Reply:
Thanks both for your explanation. Didn't realise we do not update these docs.

Reply:
I’ve reworked the document and split it in its own commit.

There’s a [live document currently circulating in the department](https://docs.google.com/document/d/1Fhu_g4Q9-a18jWftnGt1n6Scn8x88X-7jX2Ygr8LJrk/edit#), too.